src/App.jsx

```diff
@@ -1,107 +1,31 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getCommentsOfPost(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId)
+      || null
```

Comment:
```suggestion

```

Reply:
Am I need just to delete this, or remove on line 10?

Reply:
Oh, that was my main problem I suppose)